app/views/waivers/new.html.erb

```diff
@@ -1,108 +1,103 @@
+<%= form_tag( participant_waiver_path(@user) ) do %>
 
-  unenforceable or ineffective, they shall be deemed severable and all other provisions shall remain valid and binding.
-  </p>
-
-  <%= form_tag( participant_waiver_path(@user) ) do %>
   <div>
-    <%= check_box_tag :agree %> <%= label_tag :agree, "I have read this release and understand it and I voluntarily agree to all its provisions." %>
-
+    <%= check_box_tag :agree %>
+    <%= label_tag :agree, "I have read this release and understand it and I voluntarily agree to all its provisions." %>
   </div>
+
   <div>
     <%= check_box_tag :adult %>
-    <%= label_tag :adult, "I am 18 years of age or older" %>
+    <%= label_tag :adult, "I am 18 years of age or older." %>
   </div>
 
   <div>
     <h3>Emergency Alert Information</h3>
     <p>All Spring Carnival participants must provide their mobile phone number, which will be enrolled in the emergency alert system.</p>
-    <div><%= label_tag :emergency_phone, "Cell Phone:" %> <%= text_field_tag :phone_number %></div>
-    <div><%= label_tag :emergency_phone, "Phone Carrier:" %> <%= collection_select(:participant, :phone_carrier_id, PhoneCarrier.all, :id, :name) %></div>
+    <div>
+      <%= label_tag :emergency_phone, "Cell Phone:" %>
+      <%= text_field_tag :phone_number %>
+    </div>
+    <div>
+      <%= label_tag :emergency_phone, "Phone Carrier:" %>
+      <%= collection_select(:participant, :phone_carrier_id, PhoneCarrier.all, :id, :name) %>
+    </div>
   </div>
 
   <div style="margin-top: 30px;">
-    <%= submit_tag "Submit", class: "btn btn-primary" %>
+    <p>By typing your full name below you are digitally signing this document.</p>
+    <%= label_tag :signature, "Required electronic signature:" %>
+    <%= text_field_tag :signature %>
   </div>
 
-  <% end %>
-</div>
-
+  <div>
+    <%= submit_tag "Submit", class: "btn btn-primary" %>
+  </div>
 
+<% end %>
 
-<% if !current_user.participant.is_scc? or params[:participant_id]== nil %>
 <script src="https://www.youtube.com/player_api"></script>
 <script>
 
   // create youtube player
   var player;
   function onYouTubePlayerAPIReady() {
-  player = new YT.Player('player', {
-  height: '390',
-  width: '640',
-  videoId: '8ZH95wK55uY',
-  playerVars: { 'controls': 0, 'rel' : 0, 'autoplay' : 1, 'showinfo' : 0},
-  events: {
-  'onReady': onPlayerReady,
-  'onStateChange': onPlayerStateChange
-  }
-  });
+    player = new YT.Player('player', {
+      height: '390',
+      width: '640',
+      videoId: '8ZH95wK55uY',
+      playerVars: { 'controls': 0, 'rel' : 0, 'autoplay' : 1, 'showinfo' : 0},
+      events: {
+        'onReady': onPlayerReady,
+        'onStateChange': onPlayerStateChange
+      }
+    });
   }
 
-  // autoplay video
+  // autoplay video if there were no flashed errors
   function onPlayerReady(event) {
-  event.target.playVideo();
+    <%= (@should_see_video and flash[:error] == nil)? "event.target.playVideo();" : "showWaiver();" %>
```

Comment:
When testing this I couldn't get the video to skip after an error when including the `flash[:error]` check in the controller